cmd/relayproxy/api/middleware/body.go

```diff
@@ -0,0 +1,19 @@
+package middleware
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/thomaspoignant/go-feature-flag/cmd/relayproxy/config"
+	"go.uber.org/zap"
+)
+
+func BodyLogger(log *zap.Logger, cfg *config.Config) echo.MiddlewareFunc {
+	skipper := func(_ echo.Context) bool { return !cfg.IsDebugEnabled() }
+
+	return middleware.BodyDumpWithConfig(middleware.BodyDumpConfig{
+		Skipper: skipper,
+		Handler: func(_ echo.Context, req []byte, _ []byte) {
```

Comment:
```suggestion
		Handler: func(_ echo.Context, reqBody []byte, _ []byte) {
```
just to be sure this is not misleading and people knows that this does not contains the full request.